### ISSUE TITLE
websocket support

### ISSUE THREAD

**Issue by ethanaobrien** [terreng/simple-web-server#37]
simple guide - https://yaler.net/nodejs-websocket-server

The config may be difficult though.

**Comment by terreng** [comment]
Do you want to work on this? I think we should work on figuring out how this would be implemented first, and maybe save it for version 2.

I will try to be done with the UI by the end of the year.

**Comment by ethanaobrien** [comment]
Yeah, I'm planning on saving this for version 2. As at the moment I know nothing about wss servers. I just think this would be cool. 

>I will try to be done with the UI by the end of the year. I'm aiming for the initial release in January.

Cool. Thanks for letting me know.

**Comment by ethanaobrien** [comment]
@terreng I've been thinking, do you think we should do this? If the user is going to be using something like websockets chances are its going to need a more efficient, production ready (and faster) server. Do you think we should do this feature?
